frontend/src/pages/auth/signin/styles.css

```diff
@@ -0,0 +1,12 @@
+.sign-in-button {
+    background-color: #3388c7;
+    border-radius: 10px;
+    color: white;
+    font-weight: bold;
+    width: 200px;
+    margin: auto;
+}
+
+.sign-in-button:hover {
+    background-color: #0069d9;
```

Comment:
I would extract colors to a dedicated constant file colors.
As with styles, your app should look consistent and use the same colors, styles.